core/src/main/java/com/scalar/db/api/TransactionCrudOperable.java

```diff
@@ -5,7 +5,11 @@
 import java.util.List;
 import java.util.Optional;
 
-/** An interface for transaction CRUD operations. */
+/**
+ * An interface for transaction CRUD operations. Note that LINEARIZABLE consistency is always used
+ * in transactional CRUD operations, so {@link Consistency} specified for CRUD operations is
+ * ignored.
```

Comment:
Add a note that LINEARIZABLE consistency is always used in transactional CRUD operations.